README.md

```diff
@@ -68,6 +83,8 @@ easy extensions/adding Go functions to grol (see [extensions/extension.go](exten
 
 variadic functions both Go side and grol side (using `..` on grol side)
 
+Use `info` to see all the available functions, keywords, operators etc... (can be used inside functions too to examine the stack)
```

Comment:
OK would have expected to use "help"

But why not

Reply:
it's my tcl history, but more importantly, it returns a map, that can be accessed, it's not a help

we could build a help or even command completion, based on that map